spec/features/purchasing_a_reservation_on_behalf_of_spec.rb

```diff
@@ -62,6 +63,96 @@
         expect(page).to have_content "10:00 AM - 11:30 AM"
       end
     end
+  end
+
+  describe "creating multiple reservations" do
+    it "can create reservations in the future" do
+      fill_in "order[order_details][][quantity]", with: "2"
+      click_button "Create Order"
+      choose account.to_s
+      click_button "Continue"
+
+      click_link "Make a Reservation", match: :first
+      fill_in "Reserve Start", with: I18n.l(1.day.from_now.to_date, format: :usa)
+      click_button "Create"
+
+      all(:link, "Make a Reservation").last.click
+      fill_in "Reserve Start", with: I18n.l(2.days.from_now.to_date, format: :usa)
+      click_button "Create"
+
+      click_button "Purchase"
+
+      expect(page).to have_content "Order Receipt"
+      expect(page).to have_css(".currency .estimated_cost", count: 4) # Cost and Total x 2 orders
+      expect(page).to have_css(".currency .actual_cost", count: 0)
+    end
+
+    it "can create reservations in the past" do
+      fill_in "order[order_details][][quantity]", with: "2"
+      click_button "Create Order"
+      choose account.to_s
+      click_button "Continue"
+
+      click_link "Make a Reservation", match: :first
+      fill_in "Reserve Start", with: I18n.l(1.day.ago.to_date, format: :usa)
+      click_button "Create"
+
+      all(:link, "Make a Reservation").last.click
+      fill_in "Reserve Start", with: I18n.l(2.days.ago.to_date, format: :usa)
+      click_button "Create"
 
+      # this would normally be hidden and the default would be to complete backdated
+      # reservations. This might change in #121458
```

Comment:
I'm not following this comment. Doesn't this PR cover #121458?

Reply:
Oops. I meant 140462.

The "More Options" is hidden by css/js, which capybara is ignoring. But since it's visible, it'll get submitted. In that #140462 ticket, we're going to figure out what to do with past reservations but you want to select "New" as the order status.

The other option, which I'm not sure if it's worth the effort is to make these fields `disabled` by default so they don't get submitted, then enable them in the JS.